### PR TITLE
update docs to reference mermaidConfig correctly

### PR DESCRIPTION
**Pull request by antingle**
I had trouble configuring the Mermaid configuration options for this plugin because `mermaidOptions` was actually renamed to `mermaidConfig`. I only realized this when I checked the type annotations on the options object for the plugin. I have updated the docs to correctly reference the name of it, as well as update the link to the Mermaid config schema

### CODE REVIEW

**Comment by remcohaszing** [comment]
Thanks! CI is failing, but I don’t see how that’s related to your changes, so I’ll look into it.

You should probably use [`rehype-mermaidjs`](https://github.com/remcohaszing/rehype-mermaidjs) instead of this project though.

**Comment by antingle** [comment]
OK thank you. Why should I use [`rehype-mermaidjs`](https://github.com/remcohaszing/rehype-mermaidjs)? I just have a simple use case of displaying Mermaid diagrams on markdown files and this plugin does the job. My previous plugins are also remark. I believe if I want to use [`rehype-mermaidjs`](https://github.com/remcohaszing/rehype-mermaidjs), I also need to install [`rehype-remark`](https://github.com/rehypejs/rehype-remark), so that is an additional package.

**Comment by wooorm** [comment]
Depends on your use case indeed.
If you in anyway care about HTML, it’s highly likely you need rehype, and remark-rehype (not the inverse, rehype-remark).
More info here: https://github.com/rehypejs/rehype-remark#what-is-this.
I’d need to know what you’re doing to properly answer what you need

File: README.md [hunk 1]
```diff
@@ -89,11 +89,11 @@ The options used to launch the browser. (`object`)
...
 
-#### `mermaidOptions`
+#### `mermaidConfig`
 
-The [mermaid options](https://mermaid-js.github.io/mermaid/#/Setup) to use.
+The [mermaid config](https://mermaid.js.org/config/schema-docs/config.html) to use.
 
-**Note**: This options is only supported in Node.js. In the browser this option is unused. If you
+**Note**: This config is only supported in Node.js. In the browser this option is unused. If you
 use this in a browser, call `mermaid.initialize()` manually.
```
Comment (remcohaszing) [inline README.md:97]:
```suggestion
**Note**: This config is only supported in Node.js. In the browser this option is unused. If you use
this in a browser, call `mermaid.initialize()` manually.
```

**Comment by remcohaszing** [comment]
Typically people render markdown to HTML (or MDX to JS). If this is the case, you should use `rehype-mermaid`. If your goal is to replace `mermaid` code blocks in markdown files with SVG elements and then emit this as markdown, you should use `remark-mermaidjs`. This is not a common use case though.

The only reason this project isn’t deprecated, is because it is still used by [`gatsby-remark-mermaid`](https://github.com/remcohaszing/gatsby-remark-mermaid).

**Pull request merged**